### PR TITLE
S3 en and de

### PR DESCRIPTION
**Pull request by sabinem**
Provide Dataset S3 in English and German

### CODE REVIEW

File: pipelines/S3EN/S3EN.R [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    function_cofog_broad = cofog_broad,
+    institutional_sector = sector,
+    expenses_in_million_chf = mio_chf_expenditure,
+    percentage_of_total_expenses = percent_total_expenditure
+  ) %>%
+  dplyr::mutate(
+    institutional_sector = dplyr::case_when(
+      institutional_sector == "General government" ~ "Government",
+      institutional_sector == "State government" ~ "Canton",
+      institutional_sector == "Local government" ~ "Commune",
```
Comment (cmdoret) [inline pipelines/S3EN/S3EN.R:75]: Isn't the english term "Municipality"? I'd use commune in french

**Review by cmdoret — COMMENTED** [review]
Looks great! Just one minor comment on instutional sector mappings

File: pipelines/S3EN/metadata_table_columns.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name;table_name;data_type;title;title_en;example_values
+function_cofog_narrow;government_expenditure_by_function_cofog;categorical;Narrow function of govenrment by COFOG (Classification of the functions of government) of the United Nations;Narrow function of govenrment by COFOG (Classification of the functions of government) of the United Nations;"Foreign economic aid","General services","Executive and legislative organs, financial and fiscal affairs, external affairs"
+function_cofog_broad;government_expenditure_by_function_cofog;categorical;Broad function of govenrment by COFOG (Classification of the functions of government) of the United Nations;Broad function of govenrment by COFOG (Classification of the functions of government) of the United Nations;"General public services","Defence","Public order and safety"
```
Comment (cmdoret) [inline pipelines/S3EN/metadata_table_columns.csv:3]:
```suggestion
function_cofog_broad;government_expenditure_by_function_cofog;categorical;Broad function of government by COFOG (Classification of the functions of government) of the United Nations;Broad function of govenrment by COFOG (Classification of the functions of government) of the United Nations;"General public services","Defence","Public order and safety"
```

File: pipelines/S3EN/S3EN.R (hunk 1 above)
Comment (sabinem) [reply pipelines/S3EN/S3EN.R:75]: I have used these translations here: 
Screenshot 2023-10-12 at 12 39 02

They are from https://opendata.swiss/en/dataset and that belongs to the FSO, so if you don't mind I would leave it as such.

Comment (cmdoret) [reply pipelines/S3EN/S3EN.R:75]: Alright then, works for me!

**Review by cmdoret — APPROVED** [review]

**Pull request merged**